report_thesis/src/sections/results/initial_results.tex

```diff
@@ -0,0 +1,50 @@
+\subsection{Initial Results}
+As described in Section~\ref{sec:proposed_approach}, we conducted a series of initial experiments to evaluate the performance of various machine learning models on the prediction of major oxide compositions from our \gls{libs} dataset.
+These experiments aimed to provide a preliminary assessment of the models' performance.
+All models were trained on the same preprocessed data using the Norm 3 preprocessing method described in Section~\ref{sec:norm3}.
+This ensured that the models' performance could be evaluated under consistent and comparable conditions.
```

Comment:
You didn't know this, but I actually explained this detail in the experiment design. What do you think? Is it okay to summarize the point again?

Reply:
I really like Patrick's argument, but think it should be put in experiment design only. The results section is only for showing results & discussing, not presenting our approach.

Can we move it to the experiment design? And what are your thoughts @Pattrigue @Ivikhostrup?

Reply:
Perhaps a _very_ brief summary is fine to have, but not much more. The first few lines provide such a (nice) summary, and it feels like to have a 'refresher'.

But I think we need to take a pass over the sections we ref back to at some point. This one refs back to proposed approach, which is OK, but we also have a section regarding experiment design for this experiment. That should probably also get a ref, no?

Reply:
Would still like to hear thoughts on this + can someone follow up to ensure this gets done? @Pattrigue @Ivikhostrup

Reply:
This is the argument I used in the experiment design:

To conduct the initial experiment and evaluate the twelve selected models shown in Table~\ref{tab:preprocessing-models}, we ensured that the models were evaluated on a consistent basis.
We achieve this by setting the following conditions for the experiment.
The only preprocessor used for all models was \texttt{Norm 3} normalization.
This normalization technique was developed specifically by \citet{cleggRecalibrationMarsScience2017} for accomodating the characteristics of the \gls{libs} data, whereby the data represents the readings of three different spectrometers, as outlined in Section~\ref{sec:norm3}.
Therefore, \texttt{Norm 3} represents the most \gls{libs}-specific normalization technique available in from the pool of selected preprocessors and is the most appropriate choice for this experiment.

To me this sounds similar to what patrick has written here. We could take Patricks wording and replace what I have written if you want.

Reply:
In terms of referring back to the initial experiments design, I'd have to look again, but it would make sense to do so.

Reply:
Need to ensure coherence between this section and the experiment design. E.g., we don't ref back, but make the exact same point about norm 3. Defeats the purpose if we explain it again.

Just do what makes most sense here, would prefer not having to mediate that process